src/vec_buffers.h

```diff
@@ -118,6 +118,7 @@ class Buffers {
     void separate_even_odd(Buffers<T>& even, Buffers<T>& odd);
     friend bool operator==<T>(const Buffers<T>& lhs, const Buffers<T>& rhs);
     void dump(void);
+    void swap(unsigned i, unsigned j);
```

Comment:
Could be `size_t` instead of `unsigned`.

Reply:
`i` and `j` are for indices of buffers which is used as `int`. So I think `unsigned` is better.

Reply:
If they are indices, we should use `size_t`: that's the purpose of the type ("size_t is commonly used for array indexing and loop counting." from [here](https://en.cppreference.com/w/c/types/size_t) for instance)

If indices are used as `int` (but why would they: a negative index doesn't make sense to me) we shouldn't pass an `unsigned` type: mixing signed and unsigned is asking for trouble because it's easy to get subtle bug (thanks to the weird promotion rules of C and C++).

Reply:
yes, I see that. I will synchronize all indices in another commit.